frame/composable-traits/src/dex.rs

```diff
@@ -134,6 +134,31 @@ pub trait Amm {
 	) -> Result<Self::Balance, DispatchError>;
 }
 
+#[derive(Encode, Decode, MaxEncodedLen, TypeInfo, Clone, PartialEq, Eq, Copy, RuntimeDebug)]
+pub enum RewardPoolType {
+	LP,
+	PBLO,
+}
+
+impl Default for RewardPoolType {
+	fn default() -> Self {
+		RewardPoolType::PBLO
+	}
+}
+
+#[derive(
+	Encode, Decode, MaxEncodedLen, TypeInfo, Clone, Default, PartialEq, Eq, Copy, RuntimeDebug,
+)]
+pub struct StakingRewardPool<RewardPoolId> {
+	pub pool_id: RewardPoolId,
+	pub pool_type: RewardPoolType,
+}
+
+pub const MAX_REWARDS: u128 = 100_000_000_000_000_000_000_000_u128;
+pub const REWARD_PERCENTAGE: u32 = 10;
+const MILLISECS_PER_BLOCK: u32 = 12000;
+pub const FIVE_YEAR_BLOCKS: u32 = 5 * 365 * 24 * 60 * 60 / MILLISECS_PER_BLOCK;
```

Comment:
Is there any way to get the block time from the runtime?

Reply:
I tried bringing in runtime-common but that uses composable-trait as dependency so it results in dependency cycle.
I can try moving constants to pablo pallet.

Reply:
Perhaps as a Config constant?